server/container_list.go

```diff
@@ -67,13 +69,17 @@ func (s *Server) ListContainers(ctx context.Context, req *pb.ListContainersReque
 
 	for _, ctr := range ctrList {
 		if err := s.runtime.UpdateStatus(ctr); err != nil {
-			return nil, err
+			logrus.Warnf("error updating status for ctr %s: %v", ctr.ID, err)
```

Comment:
yes, we want to still show it but please refer to https://github.com/kubernetes-incubator/cri-o/pull/499#issuecomment-299860543!